go.mod

```diff
@@ -31,36 +30,36 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+replace helm.sh/helm/v3 v3.8.0 => github.com/hiddeco/helm/v3 v3.0.0-20220128105410-34ef0a7a5811
```

Comment:
Let's link to https://github.com/helm/helm/pull/10486 and explain that this replace is temporary until 3.8.1 is out.